onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -228,12 +276,32 @@ Results are from Standard_NC4as_T4_v3 Azure virtual machine:
 | torch       | 2.0.0+cu117             | compile               | 8          | 46.5            | 12,595              | 10,171               |
 | torch       | 2.0.0+cu117             | default               | 8          | 50.7            | 11,955              | 9,531                |
 
+#### Results of MI250X, 1 GCD (Ubuntu 20.04)
+
+| engine      | version                 | provider              | batch size | average latency | first run memory MB | second run memory MB |
+| ----------- | ----------------------- | --------------------- | ---------- | --------------- | ------------------- | -------------------- |
+| onnxruntime | dev                     | ROCMExecutionProvider | 1          | 2.2             | 5,548               | 4,908                |
+| torch       | 1.12.1+rocm5.4          | -                     | 1          | 3.4             | 6,653               | 4,613                |
+| torch       | 2.0.0a0+git254afb8      | default               | 1          | 3.2             | 5,991               | 4,384                |
```

Comment:
Could you test official torch 2.0 wheel?

[torch-2.0.0+rocm5.4.2-cp310-cp310-linux_x86_64.whl](https://download.pytorch.org/whl/rocm5.4.2/torch-2.0.0%2Brocm5.4.2-cp310-cp310-linux_x86_64.whl)
[torch-2.0.0+rocm5.4.2-cp39-cp39-linux_x86_64.whl](https://download.pytorch.org/whl/rocm5.4.2/torch-2.0.0%2Brocm5.4.2-cp39-cp39-linux_x86_64.whl)

Reply:
Oh, they have official wheel now. Will test it. The version is bundle with a public rocm image.

Reply:
Updated, the perf number is basically the same.